.github/workflows/test-semgrep-pro.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: |
           echo "Github event is ${{ github.event_name }}"
           if [ "${{ github.event_name }}" = "pull_request" ]; then
-            echo "docker-tag=${{ github.event.pull_request.number }}" >> "$GITHUB_OUTPUT"
+            echo "docker-tag=pr-${{ github.event.pull_request.number }}" >> "$GITHUB_OUTPUT"
```

Comment:
Is this supposed to be part of this PR?

Reply:
Please Brandon, try to cleanup your PR. Half of your PR contain unrelated changes. I don't know what's the issue, whether you wrong use git rebase or git merge, but you need to double check your PR. What I do is that each time I make a PR I read myself the PR to double check there's no thing that went in that should not.

Reply:
I usually do read the diff in GitHub, I'm not sure what happened here. I just forgot, I'm sorry.